proto/asn1-pdu/asn1_pdu.proto

```diff
@@ -43,7 +43,6 @@ message TagNumber {
 }
 
 enum LowTagNumber {
-  VAL0 = 0;
```

Comment:
I mentioned in passing Friday, but I don't think we want to remove this, because this will prevent the fuzzer from generating Context-Specific 0 (for example)

I think we need to accept that it's possible to create a single permutation that's forbidden by the encoder (Universal-0) and allow it to simply be rejected as invalid input.

Reply:
I agree with that -- we need `VAL0` to expand our input space (capture context-specific 0), even if we have to reject an invalid input. For some reason, I thought the suggestion was to take it out for now and deal with it later. Thanks for the clarification :).